kafka-ui-react-app/src/components/common/Search/__tests__/Search.spec.tsx

```diff
@@ -26,23 +26,23 @@ describe('Search', () => {
 
   describe('when placeholder is provided', () => {
     it('matches the snapshot', () => {
```

Comment:
please change the title as well, as now it says 'matches the snapshot' but you are not checking it anymore